ha/heartbeat.go

```diff
@@ -79,7 +92,17 @@ func IcingaHeartbeatListener(rdb *connection.RDBWrapper, chEnv chan *Environment
 						Name:     app.Environment,
 						ID:       Sha1bytes([]byte(app.Environment)),
 						NodeName: app.NodeName,
-						Icinga2:  Icinga2Info{app.Version, app.ProgramStart, nil},
+						Icinga2: Icinga2Info{
+							app.Version,
+							app.ProgramStart,
+							utils.EncodeChecksum(app.EndpointId),
```

Comment:
What is this change for?

Reply:
We previously didn't set the endpoint_id, because the Icinga 2 part of that was missing.